core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -192,6 +192,7 @@ public class PlanPrinter
     private final PlanRepresentation representation;
     private final Function<TableScanNode, TableInfo> tableInfoSupplier;
     private final Map<DynamicFilterId, DynamicFilterDomainStats> dynamicFilterDomainStats;
+    private final Map<PlanNodeId, Double> getSplitsTotalTime;
```

Comment:
nit: rename to `getSplitsTotalTimeNanos`. Also, make it Long if possible. `Double` for nanos is misleading.